src/main/resources/templates/smallfull/criteria.html

```diff
@@ -4,86 +4,107 @@
+
+        <ul class="govuk-list govuk-list--bullet" id="criteria-additional-criteria-list">
+          <li>turnover is not over £10.2 million</li>
+          <li>balance sheet total is not over £5.1 million</li>
+          <li>average number of employees is not over 50</li>
+        </ul>
+
+        <h2 class="govuk-heading-m" id="criteria-submit-heading">Do your prepared full accounts meet the criteria?</h2>
+
+        <div class="govuk-form-group" th:classappend="${#fields.hasErrors('isCriteriaMet')} ? 'govuk-form-group--error' : ''">
+          <fieldset class="govuk-fieldset">
+            <legend class="govuk-fieldset__legend govuk-visually-hidden">Do your prepared full accounts meet the criteria?</legend>
+
+            <span class="govuk-error-message"
+                  id="isCriteriaMet-errorId"
+                  th:if="${#fields.hasErrors('isCriteriaMet')}"
+                  th:each="e : ${#fields.errors('isCriteriaMet')}" th:text="${e}" >
+                            </span>
+
+            <div class="govuk-radios">
+              <div class="govuk-radios__item">
+                <input id="yesButton"
+                       type="radio"
+                       value="yes"
+                       th:field="*{isCriteriaMet}"
+                       th:errorclass="govuk-error-message"
+                       class="govuk-radios__input piwik-event" data-event-id="Yes"/>
+                <label class="govuk-label govuk-radios__label" for="yesButton" id="yesButton-label">
+                  Yes
+                </label>
+              </div>
+
+              <div class="govuk-radios__item">
+                <input id="noAlternativeFilingButton"
+                       type="radio"
+                       value="noAlternativeFilingMethod"
+                       th:field="*{isCriteriaMet}"
+                       th:errorclass="govuk-error-message"
+                       class="govuk-radios__input piwik-event" data-event-id="No other full"/>
+                <label class="govuk-label govuk-radios__label" for="noAlternativeFilingButton" id="noAlternativeFilingButton-label">
+                  No - show me other ways to file full accounts
+                </label>
+              </div>
+
+              <div class="govuk-radios__item">
+                <input id="noOtherAccountsButton"
+                       type="radio"
+                       value="noOtherAccounts"
+                       th:field="*{isCriteriaMet}"
+                       th:errorclass="govuk-error-message"
+                       class="govuk-radios__input piwik-event" data-event-id="No other accounts"/>
+                <label class="govuk-label govuk-radios__label" for="noOtherAccountsButton" id="noButton-label">
+                  No - I'll file other types of accounts
+                </label>
+              </div>
+
             </div>
-        </form>
+          </fieldset>
+        </div>
+
+        <div class="form-group">
```

Comment:
I followed the prototype to do all my changes so that they both were in line. I've basically copy and paste from the prototype thinking that was up-to-date and using the correct classes.

I will remove it and see if it keeps the style.